stable/jenkins/README.md

```diff
@@ -147,6 +147,8 @@ The following tables list the configurable parameters of the Jenkins chart and t
 | `master.JCasC.enabled`            | Wheter Jenkins Configuration as Code is enabled or not | `false`                 |
 | `master.JCasC.defaultConfig`      | Enables default Jenkins configuration via configuration as code plugin | `false` |
 | `master.JCasC.configScripts`      | List of Jenkins Config as Code scripts | `{}`                                    |
+| `master.JCasC.authorizationStrategy` | Jenkins Config as Code for Authorization Strategy | `loggedInUsersCanDoAnything` |
+| `master.JCasC.securityRealm`      | Jenkins Config as Code for Security Realm | `legacy`                             |
```

Comment:
Separate JCasC fields are justified since they have different default values than their XML counterparts. Using the same fields for both XML and JCasC would push the default values out of `values.yaml` and into the templates which leads to confusion as default values are expected to be in `values.yaml`.